README.md

```diff
@@ -191,7 +191,10 @@ docker-compose run -- database add-directory <DIRECTORY> [PREFIX]
 # You need to set CRATESFYI_GITHUB_USERNAME, CRATESFYI_GITHUB_ACCESSTOKEN
 # environment variables in order to run this command.
 # You can set this environment variables in ~/.cratesfyi.env file.
-docker-compose run -- database update-github-fields
+cargo run -- database update-github-fields
+# Updates github stats for crates.
+# You need to set CRATESFYI_GITLAB_ACCESSTOKEN environment variable.
+cargo run -- database update-gitlab-fields
```

Comment:
Is there any reason to make these separate commands? I'd rather have `update-repository-fields` which does both at once.

Reply:
I'll unify things a bit more. For now it allows me to test things. :)